### ISSUE TITLE
Context API

### ISSUE THREAD

**Issue by KevinMolotov** [raphamorim/react-tv#138]
The context API is broken with the React-TV-renderer. It doesn't trigger rerendering when a Consumer is subscribed to its Context.
Tried the same thing with the react-dom renderer and it's working perfectly. 

It might be because of the old version of react-reconciler? I tried to update it, but there is some major breaking changes that I can't really handle all by myself I think... Do you have any time to test this on your side @raphamorim?

**Comment by raphamorim** [comment]
yeah, probably is the old version of react-reconciler. Sure, I can take a look at this weekend. I tried to update react-ape for newest too and it's breaking some changes.
